jina/helper.py

```diff
@@ -904,7 +948,10 @@ def run_async(func, *args, **kwargs):
     """
 
     class RunThread(threading.Thread):
+        """Create a running thread when in Jupyter notebook."""
```

Comment:
This class is private to the function. Having docstrings is OK, but I'd still rename it to `_RunThread` in order to highlight, that it is private.

Reply:
Done